### PR TITLE
docs: introduces first cut of README

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Covers basic development setup only.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* link:https://godoc.org/golang.org/x/tools/cmd/goimports[`goimports`] for formatting
+* link:https://github.com/operator-framework/operator-sdk[`operator-sdk`] for code generation
+
+From now on you are ready to hack. Run `make help` to see what targets are available, but you will use
+`make build` and `make test` most often.
+
+NOTE: Have a look how link:https://github.com/moovweb/gvm[Go Version Manager] can help you simplifying configuration
+and management of different versions of Go.
+
+== License
```
Comment (aslakknutsen) [inline README.adoc]: Add LICENSE file?

Comment (bartoszmajsak) [reply README.adoc]: There's one. Can remove that.

Comment (bartoszmajsak) [reply README.adoc]: The idea was more for the generated docs site, where the file is not included.

Comment (aslakknutsen) [reply README.adoc]: hmm.. how did I miss that yesterday.. :)

**Pull request merged**